docs/includes/excel-custom-functions-note.md

```diff
@@ -1,2 +1,2 @@
 > [!NOTE]
-> Custom functions are currently available in developer preview. Custom functions are supported in Excel 2016 for Windows (build 10827 or later), Excel 2016 for Mac (build 13.329 or later), and Excel Online. Additionally, you must join the [Office Insider](https://products.office.com/office-insider) program (**Insider** level -- formerly called "Insider Fast") in order to use custom functions. Custom functions are currently disabled across all Office builds unless you are a member of the Office Insider program (**Insider** level).
\ No newline at end of file
+> Custom functions are currently available in developer preview. Custom functions are supported in Office 365 versions, Excel for Windows (Build 1810 or later), Excel for Mac (build 13.329 or later), and Excel Online. Additionally, you must join the [Office Insider](https://products.office.com/office-insider) program (**Insider** level -- formerly called "Insider Fast") in order to use custom functions. Custom functions are currently disabled across all Office builds unless you are a member of the Office Insider program (**Insider** level).
```

Comment:
What exactly are "Office 365 versions"? I guess I'm not sure how those differ from Mac/Windows/Online?

Reply:
Should read "Custom Functions are available to developers on an Office 365 subscription, or through Excel Online.  In addition to having Office 365, you must join the [Office Insider] program, and have the following builds. On Excel for Windows (Build 1810 or later), and Excel for Mac (build 13.329 or later)"

Reply:
Can you draft up that please? With the right way to write it? thanks (I think the metapoint is here Office 2016 is not correct and could be misleading)

Reply:
I spoke with @keyur32 about this earlier today to clarify my understanding of this info, and I've updated the note accordingly. Thanks, @keyur32!